packages/flutter_tools/templates/create/lib/main.dart.tmpl

```diff
@@ -84,9 +100,21 @@ class _MyHomePageState extends State<MyHomePage> {
         title: new Text(config.title),
       ),
       body: new Center(
+{{#withPluginHook}}
+        child: new Column(
```

Comment:
To clarify, I'm not suggesting forking more. That wouldn't address my concerns (it would make in fact them worse, for the reasons discussed above by @mit-mit). My concerns are not concrete, they are merely fear for the future. :-)